code/modules/admin/admin.dm

```diff
@@ -4,11 +4,11 @@ var/global/floorIsLava = 0
 var/global/enabled_spooking = 0
 
 ////////////////////////////////
-/proc/message_admins(var/msg)
+/proc/message_admins(var/msg, var/perms = (R_ADMIN|R_MOD))

```

Comment:
I'm not generally a fan. Specifically admins/mods don't have a way to distinguish this from a normal admin log atm.